plutus-ledger/src/Ledger/Tx.hs

```diff
@@ -305,7 +305,7 @@ instance Pretty CardanoTx where
                     ]) (const []) tx ++
                 [ "validity range:" <+> viaShow (getCardanoTxValidityRange tx)
                 , hang 2 (vsep ("data:": fmap pretty (Map.toList (getCardanoTxData tx))))
-                , hang 2 (vsep ("redeemers:": fmap pretty (Map.elems $ getCardanoTxRedeemers tx)))
+                , hang 2 (vsep ("redeemers:": fmap (\(k, v) -> viaShow k <+> ":" <+> viaShow v) (Map.toList $ getCardanoTxRedeemers tx)))
```

Comment:
Why `viaShow v`, `Redeemer` has a `Pretty` instance.

Reply:
One small improvement could be to remove the `Redeemer` wrapper, so it just shows `Constr 0 []`.